api/src/main/java/com/nicknathanjustin/streamercontracts/login/LoginApiController.java

```diff
@@ -33,7 +35,9 @@ public void ping(@NonNull final HttpServletRequest request,
         // Since its difficult to move the redirect, we'll have "/" function as the OauthSuccessRedirect when we see
         // a request from our Oauth login flow
         if(isOauthSuccessRedirect(auth, request)) {
-            loginService.logUserIn(auth);
+            if (!securityService.isAnonymousRequest(request)) {
```

Comment:
Why was this added and what is it doing?

Reply:
When someone sign up for our site the flow goes. frontend -> twitch -> our backend (and specifically this method)

So if this check isnt there, a blacklisted user would still be able to hit the loginService.logUserIn(auth); method and have their account added to our database